cluster/gce/configure-vm.sh

```diff
@@ -559,11 +559,8 @@ if [[ -z "${is_push}" ]]; then
   set-kube-env
   [[ "${KUBERNETES_MASTER}" == "true" ]] && mount-master-pd
   create-salt-pillar
-  if [[ "${KUBERNETES_MASTER}" == "true" ]]; then
-    create-salt-master-auth
```

Comment:
Yeah, that was a change by @ArtfulCoder to explicitly use the 10.* IP of the master (the external ipv4 address). The master can't actually talk to itself on its own 104.* address without some other network shenanigans, so a 104.* address, on the master, just won't work, unfortunately.